rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/job/RDTSyncAllLocationsServiceJob.java

```diff
@@ -0,0 +1,32 @@
+package io.ona.rdt.job;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.evernote.android.job.Job;
+
+import org.smartregister.job.SyncAllLocationsServiceJob;
+import org.smartregister.sync.intent.SyncAllLocationsIntentService;
+
+import io.ona.rdt.application.RDTApplication;
+
+public class RDTSyncAllLocationsServiceJob extends SyncAllLocationsServiceJob {
```

Comment:
Why do we need this class if changes were made in core that allow passing a custom `SyncAllLocationsService` to `SyncAllLocationsServiceJob` ?

Reply:
yes, I'll remove that